src/sage/schemes/elliptic_curves/ell_finite_field.py

```diff
@@ -1741,7 +1753,7 @@ def twists(self):
 
             sage: # needs sage.rings.finite_rings
             sage: K = GF(2**7)
-            sage: [E.ainvs() for E in EllipticCurve(j=K(1)).twists()]
+            sage: [E.ainvs() for E in EllipticCurve(j=K(1)).twists()] # random
```

Comment:
I agree with Lorenz that this could be sorted and random could be removed, or is the issue that the twists themselves are random. Maybe im being stupid here.